core/api/rpc/make.hpp

```diff
@@ -42,7 +48,14 @@ namespace fc::api {
         [&](auto &jparams,
             rpc::Respond respond,
             rpc::MakeChan make_chan,
-            rpc::Send send) {
+            rpc::Send send,
+            const rpc::Perms &perms) {
+          if (std::find(perms.begin(), perms.end(), method.getPerm())
+              == perms.end()) {
```

Comment:
this implies that `generate(admin)` wont work and should be `generate(admin, read, write)`?

Reply:
admin requires only for /remote